hosting/static/hosting/img/24-hours-support.svg

```diff
@@ -1,3 +1,20 @@
+<<<<<<< HEAD
```

Comment:
Something seems not good here. <<<<<<< HEAD - did it come from a diff tool?

Reply:
it seems there was a merge conflict for the new images, went unnoticed. Thanks for pointing out.